settings.gradle.kts

```diff
@@ -14,8 +14,6 @@ buildscript {
    }
 }
 
-ProcessBuilder("git", "submodule", "update", "--init", "--recursive").directory(settings.settingsDir).start().waitFor()

```

Comment:
It would be recommended when composite building to not init the psyonic submodule if you also have the psyonic repo cloned. otherwise the Gradle build will pick it up and ignore your other one